python/tk_desktop/desktop_window.py

```diff
@@ -818,10 +848,16 @@ def report_progress(percentage, message):
             # comes from.
             if most_recent_pipeline_configuration is None:
                 toolkit_manager.pipeline_configuration = None
+
+                # Enable user menu item to launch classic Project Setup wizard
```

Comment:
In general, pipeline configurations not using plugin ids have paths associated to them (Toolkit classic), so the configurations returned from `_get_toolkit_classic_pipeline_configurations` could has a boolean called classic. Then we could add the flag for entries coming back from `toolkit_manager.get_pipeline_configurations` before extending the list `pipeline_configurations`. Then test if the selected pipeline configuration is `not None` and doesn't have the `classic` flag. Those would be the pipeline configurations using a plugin id.

We do have an edge case where the pipeline configuration might have a plugin id AND a path for all platforms. In that case we wouldn't want to show the menu entry, but the problem here is that we are not exposing that information in get_pipeline_configurations. I think this needs to be discussed with @manneohrstrom  or over Dailies.